新版下载标签字数过多不会省略而是遮挡本子，希望改回旧版

### 复现步骤 / Step to reproduce

0.安装应用
1.进入下载页面
2.创建新标签，命名为：
[石見やそや] + [ぶーちゃん] + [ハチゴ] + [しんどう] + [ろうか] + [DOLL PLAY (黒巣ガタリ)] + [釜ボコ] + [赤月屋(赤月みゅうと)] + [極彩色 (彩社長)] + [みつ目のみつ豆 (よいころがし)] + [かえぬこ] + [つめとろ] +  [湊ゆう] + [久我繭莉] + [にのこや (にの子)] + [學園血盟帖 (織上ワト)] + [えーすけ] + [Croriin]（巨乳+很色）
3.第一个和第二个本子完全被下载标签挡死了

当我久违的更新并和往常一样下载最新版本的apk时候，发现下载标签从原本的隐藏式变成了现在的悬浮式，最终结果是我已经完全点不到最开始的2个本子了，因为已经被悬浮的标签挡的死死的


### 预期行为 / Expected behaviour

悬浮下载标签不应该阻挡本子，他应该有个字数限制，也就是说悬浮标签超过字数后以省略号代替，而不是现在这样。

### 实际行为 / Actual Behaviour

实际上他已经把我那一页下载标签的本子的最上面2本挡的死死的，怎么都点不到了。我想看甚至必须回退到很久以前的旧版本

### 应用日志 / App logs

======== PackageInfo ========
PackageName=moe.tarsin.ehviewer
VersionName=1.11.6-RC2-cc
VersionCode=180056
CommitSha=b1b8b42
CommitTime=2024-06-07 16:21

======== DeviceInfo ========
BOARD=NTH
BOOTLOADER=unknown
BRAND=HONOR
CPU_ABI=armeabi-v7a
CPU_ABI2=armeabi
DEVICE=HNNTH
DISPLAY=NTH-AN00 4.2.0.199(C00E198R2P9)
FINGERPRINT=HONOR/NTH-AN00/HNNTH:11/HONORNTH-AN00/4.2.0.199C00:user/release-keys
FINGERPRINTEX=HONOR/NTH-AN00/HNNTH:11/HONORNTH-AN00/4.2.0.199C00:user/release-keys
HARDWARE=qcom
HOST=cn-snxy1c-7f01ed4931643453248183-5b7d65f6d9-7g59b
HWFINGERPRINT=///NTH-LGRP1-CHN 4.2.0.199/NTH-AN00-CUST 4.2.0.198(C00)/NTH-AN00-PRELOAD 4.2.0.9(C00R2)//patch01
ID=HONORNTH-AN00
IS_DEBUGGABLE=false
IS_EMULATOR=false
MANUFACTURER=HONOR
MODEL=NTH-AN00
PERMISSIONS_REVIEW_REQUIRED=true
PRODUCT=NTH-AN00
RADIO=unknown
SERIAL=unknown
SUPPORTED_32_BIT_ABIS=armeabi-v7a, armeabi
SUPPORTED_64_BIT_ABIS=arm64-v8a
SUPPORTED_ABIS=arm64-v8a, armeabi-v7a, armeabi
TAGS=release-keys
TIME=1643454156000
TYPE=user
UNKNOWN=unknown
USER=test
ACTIVE_CODENAMES=
BASE_OS=
CODENAME=REL
INCREMENTAL=4.2.0.199C00
PREVIEW_SDK_FINGERPRINT=REL
PREVIEW_SDK_INT=0
RELEASE=11
RELEASE_OR_CODENAME=11
SDK=30
SDK_INT=30
SECURITY_PATCH=2022-01-01
MEMORY=33.6 MiB
MEMORY_NATIVE=23.0 MiB
MEMORY_MAX=384.0 MiB
MEMORY_TOTAL=7.2 GiB

--------- beginning of crash
05-24 05:18:35.612  1709  1709 E AndroidRuntime: FATAL EXCEPTION: main
05-24 05:18:35.612  1709  1709 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 1709
05-24 05:18:35.612  1709  1709 E AndroidRuntime: java.lang.IllegalStateException: Already resumed, but proposed with update 33
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at mk.H(Unknown Source:90)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at mk.m(Unknown Source:16)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at ED.m(Unknown Source:8)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at HD.a(Unknown Source:7)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at LD.d(Unknown Source:27)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at H.c(Unknown Source:467)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at l31.v(Unknown Source:132)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at me.m(Unknown Source:8)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at iA.b0(Unknown Source:78)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at mk.s(Unknown Source:104)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at mk.H(Unknown Source:32)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at mk.m(Unknown Source:16)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at S11.b1(Unknown Source:50)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at S11.R(Unknown Source:31)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at G.R(Unknown Source:116)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at Nt0.h(Unknown Source:156)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at Nt0.h(Unknown Source:135)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at Nt0.h(Unknown Source:135)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at Nt0.h(Unknown Source:135)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at Bk1.h(Unknown Source:35)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at lj.l(Unknown Source:137)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at n5.H(Unknown Source:80)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at n5.p(Unknown Source:213)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at n5.dispatchTouchEvent(Unknown Source:75)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3292)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2965)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at com.android.internal.policy.DecorView.superDispatchTouchEvent(DecorView.java:698)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:2030)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.app.Dialog.dispatchTouchEvent(Dialog.java:1238)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:644)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.View.dispatchPointerEvent(View.java:14757)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:7265)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:7060)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6467)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:6524)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:6490)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:6660)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:6498)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:6717)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6471)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:6524)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:6490)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:6498)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6471)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:9533)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:9435)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:9388)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:9730)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:271)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.os.MessageQueue.nativePollOnce(Native Method)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.os.MessageQueue.next(MessageQueue.java:362)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:185)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8731)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:613)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1085)
05-24 05:18:35.612  1709  1709 E AndroidRuntime: 	Suppressed: cD: [Ep0@3c3f5cd, ph@8e24e82, gZ0{Cancelling}@6d33993, W7@e2c7dd0]
05-28 12:09:51.048 22874 22874 E AndroidRuntime: FATAL EXCEPTION: main
05-28 12:09:51.048 22874 22874 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 22874
05-28 12:09:51.048 22874 22874 E AndroidRuntime: java.lang.ExceptionInInitializerError
05-28 12:09:51.048 22874 22874 E AndroidRuntime: 	at com.hippo.ehviewer.ui.MainActivity.onCreate(Unknown Source:0)
05-28 12:09:51.048 22874 22874 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:8358)
05-28 12:09:51.048 22874 22874 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:8341)
05-28 12:09:51.048 22874 22874 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1323)
05-28 12:09:51.048 22874 22874 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4058)
05-28 12:09:51.048 22874 22874 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4295)
05-28 12:09:51.048 22874 22874 E AndroidRuntime: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:93)
05-28 12:09:51.048 22874 22874 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:149)
05-28 12:09:51.048 22874 22874 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:103)
05-28 12:09:51.048 22874 22874 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2532)
05-28 12:09:51.048 22874 22874 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:109)
05-28 12:09:51.048 22874 22874 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:228)
05-28 12:09:51.048 22874 22874 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8731)
05-28 12:09:51.048 22874 22874 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-28 12:09:51.048 22874 22874 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:613)
05-28 12:09:51.048 22874 22874 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1085)
05-28 12:09:51.048 22874 22874 E AndroidRuntime: Caused by: java.lang.IllegalStateException: This should NEVER be called on the main thread! Current: Thread[main,5,main]
05-28 12:09:51.048 22874 22874 E AndroidRuntime: 	at SS0.<clinit>(Unknown Source:1285)
05-28 12:09:51.048 22874 22874 E AndroidRuntime: 	... 16 more
06-08 03:03:11.395 18917 18917 E AndroidRuntime: FATAL EXCEPTION: main
06-08 03:03:11.395 18917 18917 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 18917
06-08 03:03:11.395 18917 18917 E AndroidRuntime: java.lang.ExceptionInInitializerError
06-08 03:03:11.395 18917 18917 E AndroidRuntime: 	at com.hippo.ehviewer.ui.MainActivity.onCreate(Unknown Source:0)
06-08 03:03:11.395 18917 18917 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:8358)
06-08 03:03:11.395 18917 18917 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:8341)
06-08 03:03:11.395 18917 18917 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1323)
06-08 03:03:11.395 18917 18917 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4058)
06-08 03:03:11.395 18917 18917 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4295)
06-08 03:03:11.395 18917 18917 E AndroidRuntime: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:93)
06-08 03:03:11.395 18917 18917 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:149)
06-08 03:03:11.395 18917 18917 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:103)
06-08 03:03:11.395 18917 18917 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2532)
06-08 03:03:11.395 18917 18917 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:109)
06-08 03:03:11.395 18917 18917 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:228)
06-08 03:03:11.395 18917 18917 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8731)
06-08 03:03:11.395 18917 18917 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
06-08 03:03:11.395 18917 18917 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:613)
06-08 03:03:11.395 18917 18917 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1085)
06-08 03:03:11.395 18917 18917 E AndroidRuntime: Caused by: java.lang.IllegalStateException: This should NEVER be called on the main thread! Current: Thread[main,5,main]
06-08 03:03:11.395 18917 18917 E AndroidRuntime: 	at SS0.<clinit>(Unknown Source:1285)
06-08 03:03:11.395 18917 18917 E AndroidRuntime: 	... 16 more
06-08 03:03:11.867 19025 19025 E AndroidRuntime: FATAL EXCEPTION: main
06-08 03:03:11.867 19025 19025 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 19025
06-08 03:03:11.867 19025 19025 E AndroidRuntime: java.lang.ExceptionInInitializerError
06-08 03:03:11.867 19025 19025 E AndroidRuntime: 	at com.hippo.ehviewer.ui.MainActivity.onCreate(Unknown Source:0)
06-08 03:03:11.867 19025 19025 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:8358)
06-08 03:03:11.867 19025 19025 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:8341)
06-08 03:03:11.867 19025 19025 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1323)
06-08 03:03:11.867 19025 19025 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4058)
06-08 03:03:11.867 19025 19025 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4295)
06-08 03:03:11.867 19025 19025 E AndroidRuntime: 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:93)
06-08 03:03:11.867 19025 19025 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:149)
06-08 03:03:11.867 19025 19025 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:103)
06-08 03:03:11.867 19025 19025 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2532)
06-08 03:03:11.867 19025 19025 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:109)
06-08 03:03:11.867 19025 19025 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:228)
06-08 03:03:11.867 19025 19025 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8731)
06-08 03:03:11.867 19025 19025 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
06-08 03:03:11.867 19025 19025 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:613)
06-08 03:03:11.867 19025 19025 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1085)
06-08 03:03:11.867 19025 19025 E AndroidRuntime: Caused by: java.lang.IllegalStateException: This should NEVER be called on the main thread! Current: Thread[main,5,main]
06-08 03:03:11.867 19025 19025 E AndroidRuntime: 	at SS0.<clinit>(Unknown Source:1285)
06-08 03:03:11.867 19025 19025 E AndroidRuntime: 	... 16 more
06-09 06:37:18.102 11254 27303 E AndroidRuntime: FATAL EXCEPTION: DefaultDispatcher-worker-13
06-09 06:37:18.102 11254 27303 E AndroidRuntime: Process: moe.tarsin.ehviewer, PID: 11254
06-09 06:37:18.102 11254 27303 E AndroidRuntime: java.lang.NullPointerException
06-09 06:37:18.102 11254 27303 E AndroidRuntime: 	at lH.v(Unknown Source:152)
06-09 06:37:18.102 11254 27303 E AndroidRuntime: 	at me.m(Unknown Source:8)
06-09 06:37:18.102 11254 27303 E AndroidRuntime: 	at LE.run(Unknown Source:112)
06-09 06:37:18.102 11254 27303 E AndroidRuntime: 	at b2.run(Unknown Source:586)
06-09 06:37:18.102 11254 27303 E AndroidRuntime: 	at T31.run(Unknown Source:2)
06-09 06:37:18.102 11254 27303 E AndroidRuntime: 	at ow.run(Unknown Source:95)
06-09 06:37:18.102 11254 27303 E AndroidRuntime: 	Suppressed: cD: [gZ0{Cancelling}@21cb0b6, Dispatchers.IO]
--------- beginning of main
07-01 01:58:09.791 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:10.800 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:10.839 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:11.072 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:15.717 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:15.806 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:15.815 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:16.491 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:16.533 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:16.572 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:16.785 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:16.986 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:17.325 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:17.433 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:17.458 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:17.473 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:17.556 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:17.570 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:18.586 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:18.700 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:18.888 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:19.219 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:19.937 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:20.243 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:20.718 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:21.094 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:21.914 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:21.982 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:22.771 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:22.839 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:23.420 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:23.676 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:24.288 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:24.311 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:24.327 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:24.494 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:24.512 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:24.733 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:24.769 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:24.838 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:24.916 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:25.060 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:25.302 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:25.411 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:25.451 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:25.457 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:26.659 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:26.724 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:26.738 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:27.395 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:27.446 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:27.479 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:27.483 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:28.065 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:28.097 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:29.008 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:29.267 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:29.338 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:29.433 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:30.283 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:30.542 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:30.725 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:30.755 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:30.797 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:30.844 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:30.862 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:30.869 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:32.976 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:32.988 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:33.005 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:33.020 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:33.036 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:33.054 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:33.260 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:34.724 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:34.845 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:34.861 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:35.038 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:35.425 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:35.463 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:35.697 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:36.807 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:37.418 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:37.632 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:37.899 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:38.228 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:38.326 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:38.555 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:38.573 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:38.841 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:38.856 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:38.908 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:39.301 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:39.317 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:39.425 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:39.828 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:39.860 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:39.893 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:39.938 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:40.065 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:40.617 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:40.665 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:40.702 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:40.752 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:40.855 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:40.881 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:41.004 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:41.168 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:41.571 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:42.557 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:42.571 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:42.946 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:43.383 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:44.647 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:44.683 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:44.846 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:44.863 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:44.974 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:50.783 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:50.860 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:50.877 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:51.028 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:51.045 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:51.062 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:51.148 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:51.811 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:51.867 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:51.887 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:51.931 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:51.948 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:52.072 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:53.076 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:53.691 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:53.858 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:53.874 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:53.975 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:54.967 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:55.411 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 01:58:55.701 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:46.684 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:47.729 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:48.618 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:48.660 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:49.978 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:51.857 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:53.512 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:53.523 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:53.545 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:53.659 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:53.680 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:53.809 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:54.068 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:54.077 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:54.093 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:54.179 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:54.197 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:54.406 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:54.700 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:57.366 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:57.435 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:57.482 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:58.793 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:59.274 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:59.443 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:02:59.578 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:01.953 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:02.465 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:02.628 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:02.650 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:02.669 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:02.686 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:02.748 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:02.766 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:03.184 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:03.197 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:03.437 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:03.664 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:03.735 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:04.134 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:04.406 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:04.662 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:04.680 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:04.694 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:04.709 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:05.053 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:05.109 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:05.441 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:05.479 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:05.653 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:05.688 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:05.930 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:05.996 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:06.115 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:06.713 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:06.794 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:06.807 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:06.828 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:06.841 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:07.991 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:08.489 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:09.763 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:09.786 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:03:12.534 12888 12888 E DecorView[]: Destroying unexpected ActionMode instance of TYPE_FLOATING; com.android.internal.view.FloatingActionMode@bcab1d4 was not the current floating action mode! Expected null
07-01 02:03:15.238 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:06:29.241 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:06:30.217 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:06:30.543 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:06:30.562 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:06:31.584 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:06:31.631 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:06:31.658 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:06:31.814 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.
07-01 02:06:37.721 12888 12939 E FrameEvents: updateAcquireFence: Did not find frame.

### 备注 / Addition details

_No response_

### 浏览站点 / Browsing site

ExHentai

### EhViewer 版本号 / EhViewer version code

1.11.6-RC1

### EhViewer CI 版本 / EhViewer CI Version

6d1d737

### Android 系统版本 / Android version

Magic UI 4.2 ; Android 11

### 设备型号 / Device model

HONOR 50

### SoC 型号 / Soc model

骁龙778G

### 自查步骤 / Verify steps

- [X] 您已搜索过 [Issue Tracker](https://github.com/FooIbar/EhViewer/issues)，没有找到类似内容 / I have searched on [Issue Tracker](https://github.com/FooIbar/EhViewer/issues), No duplicate or related open issue has been found
- [X] 您确保这个 Issue 只提及一个问题。如果您有多个问题报告，烦请发起多个 Issue / Ensure there is only one bug report in this issue. Please make mutiply issue for mutiply bugs
- [X] 您确保已使用最新 [CI 版本](https://github.com/FooIbar/EhViewer/actions)测试，并且该问题在最新 CI 版本中并未解决 / This bug have not solved in latest [CI version](https://github.com/FooIbar/EhViewer/actions)

--- a/app/src/main/kotlin/com/hippo/ehviewer/ui/screen/SearchBarScreen.kt
+++ b/app/src/main/kotlin/com/hippo/ehviewer/ui/screen/SearchBarScreen.kt
@@ -227,12 +227,14 @@ fun SearchBarScreen(
                     onExpandedChange = onExpandedChange,
                     modifier = Modifier.widthIn(max = maxWidth - SearchBarHorizontalPadding * 2),
                     label = title.ifNotNullThen {
-                        Text(title!!, overflow = TextOverflow.Ellipsis)
+                        Text(title!!, overflow = TextOverflow.Ellipsis, maxLines = 1)
                     }.takeUnless {
                         val contentActive by activeState.state
                         expanded || contentActive || searchFieldState.text.isNotEmpty()
                     },
-                    placeholder = searchFieldHint.ifNotNullThen { Text(searchFieldHint!!) },
+                    placeholder = searchFieldHint.ifNotNullThen {
+                        Text(searchFieldHint!!, overflow = TextOverflow.Ellipsis, maxLines = 1)
+                    },
                     leadingIcon = {
                         if (expanded) {
                             IconButton(onClick = { hideSearchView() }) {

